### PR TITLE
Converting old acceptance tests to use new Cucumber framework

### PR DESCRIPTION
**Pull request by sebworks**
Converting old acceptance tests to use new Cucumber framework

## Additions

- Added files to convert old integration acceptance tests to use the new format.

## Changes

-  Modified `cfgov/scripts/test_data.py` to create 11 records so that  a pagination molecule would be created on the filterable pages.

- Modified `test/browser_tests/conf.js` to exclude mobile and skip tags.

## Testing

- Run `tox -e acceptance-fast` 

## Notes

- Currently  not able to test the mobile scenarios.

## Todos

- ~~Still need to pass tags from the command line.~~
- Need the ability to specify window size for headless chrome.

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by sebworks** [comment]
@richaagarwal, please test this when you get a chance. I've modified the code to fix issues with running in headless mode while passing the mobile tag.  You can test using: 

`gulp test:acceptance --fast --tags=@mobile`

File: gulp/utils/handle-errors.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ module.exports = function() {
                    && this.tasks.browsersync
                    && this.tasks.browsersync.done === false;
 
+  console.log( 'isWatching', isWatching );
```
Comment (richaagarwal) [inline gulp/utils/handle-errors.js]: Can this be removed now?

File: test/browser_tests/util/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function shouldShouldnt( value ) {
```
Comment (richaagarwal) [inline test/browser_tests/util/index.js]: I don't understand this naming or what it's doing. Could you explain?

File: gulp/utils/handle-errors.js (hunk 1 above)
Comment (sebworks) [inline gulp/utils/handle-errors.js]: Yeah, removed in #3005.

File: test/browser_tests/util/index.js (hunk 2 above)
Comment (sebworks) [inline test/browser_tests/util/index.js]: It just converts `('should/shouldn't)` to boolean. I've added some documentation https://github.com/cfpb/cfgov-refresh/pull/3005/commits/0cc4b04f49867c50f8e0283ca3ed8b60bfc71664.

**Pull request merged**